### ISSUE TITLE
Some Cesium logos have incorrect colors

### ISSUE THREAD

**Issue by hpinkos** [AnalyticalGraphicsInc/cesium#4607]
I don't know how I didn't notice this when I was reviewing #4584, but we have two different color logos on https://github.com/AnalyticalGraphicsInc/cesium/wiki/Cesium-Logos

![logo1](https://github.com/AnalyticalGraphicsInc/cesium/wiki/logos/Cesium_Logo_Color.jpg)
![logo2](https://github.com/AnalyticalGraphicsInc/cesium/wiki/logos/Cesium_Logo_Color_1.png)

In case you can't see it:
![image](https://cloud.githubusercontent.com/assets/3451886/20021834/11171250-a2ae-11e6-8a25-8d3f4aaf2590.png)

The first one, the darker green, matches the RGB values on the Wikipedia page.  Is that the correct one?
The lighter green is used in the SVG we have.

I'm fine with using either but we need to decide on one and use the same colors everywhere.

**Comment by mramato** [comment]
Probably related to #3847?

**Comment by hpinkos** [comment]
The R value for the blue is wrong there, so I wouldn't say this is directly related.  But both this and #3847 can probably be fixed at the same time

**Comment by emackey** [comment]
Great eye @hpinkos.  I just had a long offline chat with Jason Martin about this.  He looked up the official colors for us.

The official blue is `rgb(109, 171, 228)` `#6dabe4`, and the official green is `rgb(122, 156, 73)` `#7a9c49`.  The good news is, this matches exactly what the SVG has on the wiki.  I used this SVG to build all of the new logos in #4584, so those are the correct colors too.

But, some of the older logos on the wiki are bad, and the style guide is no good (wrong RGB and wrong CYMK values, and a typo in an RGB value).  The "Spot Pantone" colors in the style guide are the only ones in that guide that are correct (Blue is PMS 284, green is PMS 576).

Also there happened to be a large officially printed Cesium logo near Jason's cube, and I confirmed it looked visually more like the SVG and less like the darker green from the old style guide.
